CHANGES.md

```diff
@@ -3,10 +3,18 @@ Change Log
 
 ### 1.9 - 2015-05-01
 * Breaking changes
-  *
+  * Removed `ColorMaterialProperty.fromColor`. Pass a `Color` directly to the `ColorMaterialProperty` constructor instead.
```

Comment:
I partially agree with you, but I think on the whole we need a firmer deprecation policy.  If it's always 3 releases, we can just say it once at the top, i.e. "Breaking changes that were previously deprecated in 1.6" or something like that.  For now, I'll just add it to each item.